apps/tutorial_servo_control/main.cpp

```diff
@@ -0,0 +1,68 @@
+#include "mbed.h"

```

Comment:
the training has 2 different challenges. Using a potentiometer to control a servo and using CAN bus to control the servo. these should be different apps. it looks like you combined them into one. we can have a call if the instructions arent clear